bitnami/appsmith/templates/backend/deployment.yaml

```diff
@@ -79,6 +79,8 @@ spec:
           securityContext:
             runAsUser: 0
           resources: {{- toYaml .Values.volumePermissions.resources | nindent 12 }}
+          {{- else if ne .Values.volumePermissions.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.volumePermissions.resourcesPreset) | nindent 12 }}
```

Comment:
Missing `{{- if .Values.volumePermissions.resources }}` in line 81, and `{{- end }}` in line 84.